site/src/docs/getting-started/tutorials/figma-tutorial.mdx

```diff
@@ -0,0 +1,81 @@
+
+<Image src="/images/getting-started/tutorials/figma-basics/figma-create-new-project.png" alt="Adding a project description in Figma" style={{ display: 'block', width: '100%', minWidth: 600, maxwidth: 915, margin: '0 auto' }} viewable/>
+
+Next, input a descriptive name for your project. 
+
+Lastly, open your newly created project and write a description for it by selecting "Add description" in the dropdown next to your project name. It is recommended to briefly describe the goal of the project and also provide contact information of the people responsible for the design files.
+
+Now you are ready to start using Figma and to create your first design files.
+
+### How to use the HDS components in a project?
+To be able to use HDS components in your design files, you need to link the design libraries to the file. In most cases if you are inside City of Helsinki organisation, HDS Design kit is automatically visible in your project’s libraries. If it is not to link the HDS Design kit, open the project. Browse to the Figma main menu and choose “Libraries” 
+
+<Image src="/images/getting-started/tutorials/figma-basics/figma-main-menu.png" alt="Main menu button Figma" style={{ display: 'block', width: '100%', minWidth: 600, maxwidth: 915, margin: '0 auto' }} viewable/>
+
+In the library menu that pops up, under Helsinki Design system-team find “HDS Design kit vX.X.X” and press the “Add to file” button. 
+
+<Image src="/images/getting-started/tutorials/figma-basics/figma-add-libraries.png" alt="Figma add Design kit library to your file" style={{ display: 'block', width: '100%', minWidth: 600, maxwidth: 915, margin: '0 auto' }} viewable/>
+
+Now HDS Design kit library is added to the project and components and styles show up in the project's “Assets” menu.
+
+### How to use HDS components in Figma?
+To use HDS components in your Figma files, in Figma open up the "Assets" menu on top of the left panel. In this menu, you can type to search HDS components or select a specific component on the left-hand panel. Double click the component or drag it to add it to your file.
+
+If you do not see HDS components in the "Assets" menu, please make sure you have HDS Design kit library added to your file. See the [above question "How to use the HDS components in a project?"](#how-to-use-the-hds-components-in-a-project)
+
+### How to update HDS libraries?
+When HDS design libraries update, and you have used updated components in your project, you will receive a small notification "Component Updates Available" at the bottom-right corner of the Figma window. Clicking the notification opens a window in which you can review the changes the HDS has made to the components. You can also view updates in the “Assets” menu library tab.
```

Comment:
This has been updated lately. Figma will now notify about library updates in the top toolbar.

> "When updates are available for a style, component, or variable **on the current page**, the  Libraries icon in the top toolbar will display a blue badge."

**Here's my suggestion:** _When HDS design libraries update, and you have used updated style, component, or variable in your project on the currently open page, the Libraries icon in the top toolbar will display a blue badge. Clicking the notification opens a window where you can review the changes that HDS has made to the assets on the current page. To view pending updates for assets across all pages, click 'All pages' tab.'_

I'd also consider using assets (or "style, component, or variable") rather than just a component when describing all Figma library content.

https://help.figma.com/hc/en-us/articles/360039234193-Review-and-accept-library-updates